src/m365/tenant/commands/status/status-list.spec.ts

```diff
@@ -62,13 +61,12 @@ describe(commands.TENANT_STATUS_LIST, () => {
     sinon.stub(auth, 'restoreAuth').callsFake(() => Promise.resolve());
     sinon.stub(appInsights, 'trackEvent').callsFake(() => { });
     auth.service.connected = true;
-    auth.service.spoUrl = 'https://contoso.sharepoint.com';
+    auth.service.tenantId = '48526e9f-60c5-3000-31d7-aa1dc75ecf3c|908bel80-a04a-4422-b4a0-883d9847d110:c8e761e2-d528-34d1-8776-dc51157d619a&#xA;Tenant';
```

Comment:
This value is not used in tests, so there is no need to set it. Instead, you should have either define a mock access token to be used by the `Utils.getTenantIdFromAccessToken` method or mock the call to `Utils.getTenantIdFromAccessToken` to return a tenant ID.

Reply:
Hi @waldekmastykarz 
Are you getting below error for failed tests?
`Timeout of 2000ms exceeded. For async tests and hooks, ensure "done()" is called; if returning a Promise, ensure it resolves.`

Reply:
Yes, that's the same issues I'm seeing

Reply:
Hi @waldekmastykarz 
I am looking in to the cause of this error, but not sure of cause of this. The test case pattern is same as like other test cases. 🤔
Do you have any pointers?